app/test/unit/git/log-test.ts

```diff
@@ -34,11 +34,10 @@ describe('git/log', () => {
       const path = await setupFixtureRepository('just-doing-some-signing')
       const repository = new Repository(path, 1, null, false)
 
-      // ensure the test repository is configured to detect copies
-      await GitProcess.exec(
-        ['config', 'log.showSignature', 'true'],
-        repository.path
-      )
+      // ensure the default config is to try and show signatures
+      // this should be overriden by the `getCommits` function as it may not
+      // have a valid GPG agent configured
+      await setupLocalConfig(repository, [['log.showSignature', 'true']])
```

Comment:
The comment attached to this setup wasn't even remotely accurate to what was being tested, so I updated that to better explain this test case.